src/js/form.js

```diff
@@ -923,24 +927,6 @@ Form.prototype.blockPageNavigation = function() {
     }, 600 );
 };
 
-/**
- * Checks whether the question is not currently marked as invalid. If no argument is provided, it checks the whole form.
- *
- * @param {Element} node - form control HTML element
- * @return {!boolean} Whether the question/form is not marked as invalid.
- */
-Form.prototype.isValid = function( node ) {
-    const invalidSelectors = [ '.invalid-required', '.invalid-relevant' ].concat(  this.constraintClassesInvalid.map( cls => `.${cls}` ) );
-    if ( node ) {
-        const question = this.input.getWrapNode( node );
-        const cls = question.classList;
-
-        return !invalidSelectors.some( selector => cls.contains( selector ) );
```

Comment:
This always returns true because `selector` should a className without a dot prefixed to it.